cmd/libsnap-confine-private/snap-test.c

```diff
@@ -118,13 +118,13 @@ static void test_sc_is_hook_security_tag(void)
 	g_assert_true(sc_is_hook_security_tag
 		      ("snap.foo_instance.hook.bar-baz"));
 	g_assert_true(sc_is_hook_security_tag("snap.foo_bar.hook.bar-baz"));
+	g_assert_true(sc_is_hook_security_tag("snap.foo_bar.hook.f00"));
```

Comment:
`snap.foo_bar.hook.f-0-0` would now be considered valid too

Reply:
Indeed, added a test case as well. FWTW it matches snapd's regexp and our validPlugSlotIface also allows plugs/slots like that, so it needs to stay that way not to break compatibility.